src/frontend/src/routes/router.tsx

```diff
@@ -44,6 +45,7 @@ const Router = () => {
         <Route path="/pia/intake/:id/:title" element={<PIADetailPage />} />
       </Route>
       <Route path="/not-authorized" element={<Unauthorized />} />
+      <Route path="/tab2" element={<PIAAdditionalRisks />} />
```

Comment:
same as last pr, this route just for test purpose, I will delete it before merge the pr to main